src/utils/ruleCalls.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const listRules = async () => {
+  const response = await axios.get(process.env.API_URL);
```

Comment:
cool for using environment variable! later on you can do stuff like `${baseAPIUrl}/otherRestfulRoute and other stuff for your api calls